apache-maven/src/assembly/shared/init

```diff
@@ -20,7 +20,7 @@ if [ -n "$JAVA_HOME" ] ; then
     fi
   fi
 else
-  JAVACMD="`which java`"
+  JAVACMD="`\\unalias -a; unset -f command; command -v java`"
```

Comment:
* Do you assume as `unalias` could be an alias as well?
* Why are you unset all alias? Do you assume `java(1)` to be an alias?